pennylane/wires.py

```diff
@@ -495,15 +495,15 @@ def unique_wires(list_of_wires):
         return Wires(tuple(unique), _override=True)
 
     def union(self, other):
-        """Return the union of the current Wires object and either another Wires object or an
-        iterable that can be interpreted like a Wires object e.g., List.
+        """Return the union of the current :class:`~.Wires` object and either another :class:`~.Wires` object or an
+        iterable that can be interpreted like a :class:`~.Wires` object e.g., a ``List``.
 
         Args:
-            other (Any): Wires or any iterable that can be interpreted like a Wires object
-                to perform the union with. See _process for details on the interpretation.
+            other (Any): :class:`~.Wires` or any iterable that can be interpreted like a :class:`~.Wires` object
+                to perform the union with. See ``_process`` for details on the interpretation.
```

Comment:
I know this was the case before this PR already, but it seems a bit strange to me to refer to a private function for details in a public docstring. We should move any relevant details into a public docstring, most likely the present one.